src/string_util.cc

```diff
@@ -5,6 +5,7 @@
 #include <array>
 #include <memory>
 #include <sstream>
+#include <stdio.h>
```

Comment:
I added stdio.h because it's vsnprintf that is used, not std::vsnprintf.
Should it still be cstdio?